pom.xml

```diff
@@ -66,6 +66,10 @@
 			<artifactId>liquibase-maven-plugin</artifactId>
 			<version>${liquibase.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
```

Comment:
IMO it's better to add spring starter instead of concrete implementation

Reply:
Ok. Replaced in the next committee